app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/protectiveword/ProtectiveWordFragment.kt

```diff
@@ -34,6 +36,22 @@ class ProtectiveWordFragment : Fragment(R.layout.fragment_protective_word) {
     private val viewModel: ProtectiveWordViewModel by viewModels()
     private val args: ProtectiveWordFragmentArgs by navArgs()
     private val sharedViewModel: SharedViewModel by activityViewModels()
+    private val textWatcher = object : TextWatcher {
```

Comment:
You can use edtText extension method for the task

Reply:
using extension methods also gave same challenge as I was trying to set the text in the listener methods. Used input filter in this case.